packages/image_picker/image_picker_for_windows/pubspec.yaml

```diff
@@ -0,0 +1,28 @@
+name: image_picker_for_windows
+description: Windows platform implementation of image_picker
+repository: https://github.com/flutter/plugins/tree/main/packages/image_picker/image_picker_for_windows
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+image_picker%22
+version: 0.0.1
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=2.0.0"
+
+flutter:
+  plugin:
+    implements: image_picker
+    platforms:
+      windows:
+        dartPluginClass: ImagePickerWindowsPlugin
+
+dependencies:
+  flutter:
+    sdk: flutter
+  image_picker_platform_interface: ^2.4.3
+  file_selector: ^0.8.3
```

Comment:
This should be `file_selector_platform_interface`; we don't want to pull in the native `file_selector` implementations for other platforms and bloat people's apps unnecessarily.